templates/theme/index.json-ld

```diff
@@ -0,0 +1,203 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Theme",
+		"fr": "Thème"
+	},
+	"description": {
+		"en": "Theme page template",
+		"fr": "Gabarit de page thème"
+	},
+	"modified": "2020-02-12",
+	"componentName": "theme",
+	"status": "stable",
+	"version": "1.0",
```

Comment:
Set this as the version 2.0.0 

No need to document the version 1.0, but as the migration instruction you can say that the whole page must be re-written (or have this as an instruction note)